docs/docs/migration-guide.mdx

```diff
@@ -10,6 +10,62 @@ description: |
 This page contains information about changes between major versions and
 how you can migrate from one version to another.
 
+## Rasa 2.5 to 2.6
+
+### Forms
+
+#### New `ignored_intents` parameter in Forms
+
+We added a new parameter under Forms with the name `ignored_intents`. This parameter
+can be used in case the user never wants to fill any slots of a form with the specified
+intent, e.g. chitchat. Please see the [Forms documentation](forms.mdx) for examples and
+more information on how to use it in your `domain.yml` file.
+
+Before, if a user did not want to fill any slots of a form with a specified intent
+they would have to define it under the `not_intent` parameter for every slot mapping
+as shown in the following example :
+
+```yaml
+forms:
+  restaurant_form:
+    required_slots:
```

Comment:
Yes, I thought about this but I added the `required_slots` in all examples, tests, docs, everywhere so I thought it would be weird to only have this place that `required_slots` keyword is missing. But ok, I removed it.